### DISCUSSION TITLE
Unlockables being reverted back to prior state

### DISCUSSION THREAD

**Discussion by Leroy8888** [HelixNebulaStudio/RiseOfTheDead#354] [Q&A] [answered]
I noticed a bit ago that in the dev branch logs the unlockables go reverted back to normal.

I am not against the change at all, but I am curious to hear the decision for its implementation and its removal shortly after.

**Comment by MXKhronos** [comment, marked as answer]
I answered it here https://github.com/HelixNebulaStudio/RiseOfTheDead/discussions/355
